cinch/site.yml

```diff
@@ -1,8 +1,13 @@
 - name: wait for SSH
   hosts: all
   gather_facts: false
-  roles:
-    - check_ssh
+  tasks:
```

Comment:
This should simply have replaced the body of the `check_ssh` role. `tasks` should be avoided in top level playbooks, especially those with multiple parameters that might be (in this case, are) used repeatedly.

Reply:
Overall it is considered best practices to wrap all tasks inside of rolls. Especially if those rolls could be useful to others down the line. Cinch is not currently configured to use or consume rolls from Ansible Galaxy, but I'd love for that to change in the future so it can get on board with the [oasis](https://github.com/oasis-roles/) project. Waiting for SSH is definitely one of those highly sought after and reusable components.

Reply:
@greg-hellings  OK. Just sometimes I am confused by whether to wrap a **single task** into a role or just put it to the top level. From my view, it's a little bit redundant to make it a role. But since the check_ssh have some params, I think it's OK.